app/app.js

```diff
@@ -428,7 +428,7 @@ angular.module('portainer', [
       }
     })
     .state('service', {
-      url: '^/service/:id/',
+      url: '^/service/:id/:page_anchor',
```

Comment:
Ok have a look at https://github.com/portainer/portainer/commit/b42272cd1223634133f67b2e818f15fd72b6c809

This is an example on how to use `$anchorScroll()` to automatically scrolls to element with IDs.

This way you can use: http://portainer.domain:9000/#/service/ID/#networks to access networks for example.

And then you can use `$location.hash('networks')` to scroll to the network element programmatically.